File: Beecrowd/Beginner/1000 - 1099/1064 - Positives and Average/C/1064_PositivesandAverage.c
#include <stdio.h>

int main() {
    double number, sum = 0;
    int count = 0;

    for (int i = 0; i < 6; i++) {
        scanf("%lf", &number);

        if (number > 0) {
            count++;
            sum += number;
        }
    }

    printf("%d valores positivos\n", count);
    printf("%.1lf\n", sum / count);

    return 0;
}

File: Beecrowd/Beginner/1000 - 1099/1064 - Positives and Average/Python/1064_PositivesandAverage.py
count = 0
sum_val = 0

for i in range(6):
    number = float(input())

    if number > 0:
        count += 1
        sum_val += number

print(f"{count} valores positivos")
if count > 0:
    print(f"{sum_val / count:.1f}")
